app/models/user_profile.rb

```diff
@@ -126,7 +125,6 @@ def website_domain_validator
 #  bio_cooked_version   :integer
 #  badge_granted_title  :boolean          default(FALSE)
 #  card_background      :string(255)
-#  card_image_badge_id  :integer
```

Comment:
I think it would be a good idea to add `self.ignored_columns = %w(card_image_badge_id)` to this model.

Reply:
Yes that's what I missed. Thanks!